src/uct/api/uct.h

```diff
@@ -3090,6 +3090,21 @@ ucs_status_t uct_cm_config_read(uct_component_h component,
                                 uct_cm_config_t **config_p);
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Notify the server about client-side connection establishment.
+ *
+ * This routine should be called on the client side after the client completed
+ * establishing its connection to the server. The routine will send a
+ * notification message to the server indicating that the client is connected.
+ *
+ * @param [in]  ep      The connected endpoint on the client side.
+ *
+ * @return Error code.
+ */
+ucs_status_t uct_cm_client_ep_conn_notify(uct_ep_h ep);
```

Comment:
@alinask Does `uct_cm_ep_server_notify_callback_t` must  call `uct_cm_client_ep_conn_notify` to notify the server ?

Reply:
yes

Reply:
@yosefe let's explicitly document this in this function. Essentially the callback has to use this function to notify the server and complete connection setup. Otherwise the connection is incomplete, etc.

Reply:
@shamisp the `uct_cm_client_ep_conn_notify` function should be called on the client side, when the client completes its connection establishment to the server. This function will send a notification to the server and once the server receives this notification, it should invoke its `uct_cm_ep_server_notify_callback_t` callback to complete the connection establishment on its side.
This is documented on both sides. Do you think it needs enhancement? what is missing?

Reply:
to add that notify() api should be called on client side at uct.h:175